src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -125,13 +125,13 @@ private void updatePreferences() {
       preferences.putAll(form.getConfPanel().getConfiguration().asMap());
 
     // Clean all custom conf keys
-    forms.forEach(formId ->
-        preferences.removeAll(ExportConfiguration.keys(buildCustomConfPrefix(formId)))
+    forms.forEach(formStatus ->
```

Comment:
`forEach` here is a misnomer, wouldn’t you say? I don’t expect a for each to first map to get a specific property.